CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Changed
+- Bump `eth-block-tracker` from `^4.4.3` to `^8.1.0` ([#56](https://github.com/MetaMask/nonce-tracker/pull/56))
```

Comment:
Is this breaking, and if so, why?

Reply:
Ah. Well, I should have provided more detail in the PR itself, so that's my bad. That said, but we have been writing changelogs with the thought that we _shouldn't_ expect consumers to have to follow PRs for critical information. I know it makes my job easier whenever I'm upgrading a dependency, at least.

Looking back through the releases, it does look like the addition of TypeScript types is the thing that makes this breaking (all of the other changes, including the Node upgrade, have either already been made, or do not apply). So, can we say this?

``` suggestion
- **BREAKING:** Bump `eth-block-tracker` from `^4.4.3` to `^8.1.0` ([#56](https://github.com/MetaMask/nonce-tracker/pull/56))
  - When creating a new NonceTracker, you must pass in an instance of PollingBlockTracker (failure to do so now produces a type error).